src/newcharacter.cpp

```diff
@@ -4547,7 +4547,11 @@ void set_description( tab_manager &tabs, avatar &you, const bool allow_reroll,
         } else if( action == "RANDOMIZE_CHAR_DESCRIPTION" ) {
             bool gender_selection = one_in( 2 );
             you.male = gender_selection;
-            outfit = gender_selection;
+            if( one_in( 10 ) ) {
```

Comment:
You're not going for a demographically accurate probability here, I imagine?

Reply:
We don't really need to base this on reality, it doesn't affect NPCs or any other thing than the player, in the case of making a random character.

Reply:
If you have demographics for us, feel free to provide them for context if nothing else.
I ran across this, but it's hard to draw conclusions from it https://www.statista.com/chart/30170/respondents-that-have-purchased-fashion-outside-of-their-gender-identity/